### PR TITLE
Update WG_NAME_graduation_stage.md

### PR DESCRIPTION
**Pull request by SecurityCRob**
suggested updates to WG graduated template

### CODE REVIEW

File: process/templates/WG_NAME_graduation_stage.md [hunk 1]
```diff
@@ -1,11 +1,36 @@
...
+  * "link to Incubating PR"
+
 ### List of regular contributors
 The WG must have at least 5 contributors from at least 3 different organizations attending regularly as recorded in meeting minutes.
-
   * "name, affiliation, GitHub ID"
 
-### Working Group activity
-The WG must have met at least 4 times over a period of at least 2 months since becoming `Incubating`
+### Release cadence
```
Comment (marcelamelara) [inline process/templates/WG_NAME_graduation_stage.md:10]: What does "Release" mean in the context of WGs?

File: process/templates/WG_NAME_graduation_stage.md [hunk 2]
```diff
@@ -1,11 +1,36 @@
...
+The TI must be able to show a consistent release cadence.
+  * "link to release documentation"
+
+### Governance
```
Comment (marcelamelara) [inline process/templates/WG_NAME_graduation_stage.md:14]: Looks like these were copied over from the Projects doc?

File: process/templates/WG_NAME_graduation_stage.md (hunk 1 above)
Comment (SecurityCRob) [reply process/templates/WG_NAME_graduation_stage.md:10]: the WG should have some activities and artifacts that are shared publicly from time to time.  Publishing guides, whitepapers, blogs, something that furthers our missions and goals.

File: process/templates/WG_NAME_graduation_stage.md (hunk 2 above)
Comment (SecurityCRob) [reply process/templates/WG_NAME_graduation_stage.md:14]: it was, but the idea is the same:  we want our WGs to continue to collaborate and work together and periodically deliver new or updated things to the community

File: process/templates/WG_NAME_graduation_stage.md [hunk 3]
```diff
@@ -1,11 +1,41 @@
 ## Working Group graduation application
 
+### WG has met all Incubating requirements
+  * "link to Incubating PR"
+
+### List WG Chair(s) and or Vice Chair
+The WG must have a minimum of 2 Chair(s) or 1 Chair and 1 Vice Chair
+
+  * "name, affiliation, GitHUb ID"
+
```
Comment (steiza) [inline process/templates/WG_NAME_graduation_stage.md:10]:
```suggestion
```

This is an interesting idea for the future, but for now we're going to not add new requirements until after we land these docs.

File: process/templates/WG_NAME_graduation_stage.md [hunk 4]
```diff
@@ -1,11 +1,41 @@
...
 The WG must have at least 5 contributors from at least 3 different organizations attending regularly as recorded in meeting minutes.
-
   * "name, affiliation, GitHub ID"
 
-### Working Group activity
-The WG must have met at least 4 times over a period of at least 2 months since becoming `Incubating`
+### Release cadence
+The TI must be able to show a consistent release cadence.
+  * "link to release documentation"
+
```
Comment (steiza) [inline process/templates/WG_NAME_graduation_stage.md:18]:
```suggestion
```

I think this is a good idea that we should discuss further (see for example https://repos.openssf.org/), but it is not currently a requirement today.

File: process/templates/WG_NAME_graduation_stage.md [hunk 5]
```diff
@@ -1,11 +1,41 @@
...
+### Governance
+Projects must have documented project governance and be able to demonstrate that governance in action.
+  * "link to governance documents"
+Projects have met at least 4 times over a period of at least 2 months since becoming incubating
+  * "link to meeting agenda"
+Have a define roadmap and annual goals for the project
+  * "link to roadmap and goals"
```
Comment (steiza) [inline process/templates/WG_NAME_graduation_stage.md:25]:
```suggestion
```

This is a good idea but not a requirement today.

File: process/templates/WG_NAME_graduation_stage.md [hunk 6]
```diff
@@ -1,11 +1,41 @@
...
 
-### Working Group activity
-The WG must have met at least 4 times over a period of at least 2 months since becoming `Incubating`
+### Release cadence
+The TI must be able to show a consistent release cadence.
+  * "link to release documentation"
+
+### Governance
+Projects must have documented project governance and be able to demonstrate that governance in action.
+  * "link to governance documents"
```
Comment (steiza) [inline process/templates/WG_NAME_graduation_stage.md:21]:
```suggestion
```

This is a good idea but not a requirement today.

File: process/templates/WG_NAME_graduation_stage.md [hunk 7]
```diff
@@ -1,11 +1,41 @@
...
+Projects have met at least 4 times over a period of at least 2 months since becoming incubating
+  * "link to meeting agenda"
+Have a define roadmap and annual goals for the project
+  * "link to roadmap and goals"
+
+### To retain Graduated status
+A TI is expected to continue operating per the above guidelines, and to provide the TAC with quarterly status updates, and to approach the TAC when seeking approval of substantial changes (such as when accepting or promoting new Projects).
```
Comment (steiza) [inline process/templates/WG_NAME_graduation_stage.md:28]:
```suggestion
```

This is true, but is covered by https://github.com/ossf/tac/blob/main/process/working-group-lifecycle.md and doesn't need to be part of the application process (and we don't have a section like this on the working group sandbox or incubating template).

Comment (steiza) [reply process/templates/WG_NAME_graduation_stage.md:15]: @SecurityCRob are you still thinking about this change? Or do we want to leave it in?

Comment (steiza) [reply process/templates/WG_NAME_graduation_stage.md:15]: From https://github.com/ossf/tac/pull/279#discussion_r1523586318 it sounds like we'd like to keep these.

**Review by steiza — APPROVED** [review]
Thanks!

**Review by mlieberman85 — APPROVED** [review]
LGTM

**Review by sevansdell — APPROVED** [review]
looks good.

**Comment by SecurityCRob** [comment]
5 of 9 tac members approved, merging

**Pull request merged**